### ISSUE TITLE
Downgrading snapchat

### ISSUE THREAD

**Issue by niober** [JorgenPhi/php-snapchat#104]
Okey I think you will be really interested in what I have to say here. I have ALMOST solved the whole snapchat downgrade problem. I can tell you all about how but first in need someone's help here. Now I have successfully installed snapchat 6.1.1 on my jailbroken iPhone 4 ios 7.1.2. When I open snapchat 6.1.1 there is no crash and I can successfully watch my friends story's and also load story's. The problem is that I can't upload story's, I can't load snaps that I got and I can't send snaps either. So If someone know how to fix that problem then I think we have solved the whole problem;)

**Comment by mgp25** [comment]
https://github.com/mgp25/SC-API

**Comment by niober** [comment]
What do you mean with that link?

**Comment by mgp25** [comment]
@niober updated SnapChat API, working on latest version with almost all features, there is no need to downgrade API version

Regards

**Comment by hako** [comment]
@niober Its where all the development of the new PHP Snapchat API is. Not here, php-snapchat is too old.

**Comment by mgp25** [comment]
@hako good to see you here, if you have android device, could you sniff the request? i think it has changed at least some params

**Comment by niober** [comment]
I don't use that API I just thought that you maybe could help me with my problem that I told you above. Do you now how to fix it cause my goal is to successfully downgrade snapchat to under version 9. In case you don't understand read what I wrote first

**Comment by niober** [comment]
Just help me to fix the problem that I can see stories but I can't upload stories or send snaps in the old version of snapchat.

**Comment by mgp25** [comment]
@niober what's the point of  installing / having a downgraded version? And by the way, this is an 3rd party API not Snapchat app support, better send them an email and explain them your problem.

https://www.snapchat.com/

Regards

**Comment by hako** [comment]
@mgp25 Which endpoints in particular?

**Comment by niober** [comment]
Many, MANY people want to downgrade snapchat cause of bad updates.... And I can't contact anapchat support cause it is not okey to downgrade snapchat. I just thought you guys could help me with this snapchat server problem.

**Comment by mgp25** [comment]
@hako `/loq/register` i think, with the actual version it gives me issues while trying to register and that's why people is getting an empty zip package for the captcha

**Comment by niober** [comment]
Please help me

**Comment by mgp25** [comment]
@niober instead of downgrading, you can always disable the updates you don't like with simple tweaks, you have a jailbroken device.

Note that maybe some features won't work anymore in previous Snapchat versions

**Comment by niober** [comment]
Yeah but It must go to fix it cause previous version of snapchat has worked once so it must go to fix it again
